docs/data/charts/components/components.md

```diff
@@ -79,29 +81,25 @@ By using `invert`, the value associated with the current mouse coordinate `y` ca
 
 ### Series
 
-Series information is accessible through the `useSeries` hook for all series types, and `useXxxSeries` hook for a specific series type.
-These hooks return the order of the series and their configuration, including data points, color, among others.
+Series information is accessible through the `useSeries()` hook for all series types, and the `useXxxSeries()` hook for a specific series type.
```

Comment:
@mui/xcharts I believe we started using `use*Series` from the discussion here: https://github.com/mui/mui-x/pull/19334#discussion_r2355682901

```suggestion
Series information is accessible through the `useSeries()` hook for all series types, and the `use*Series()` hook for a specific series type.
```

Reply:
I missed this comment. For me `use*Series()` is not intuitive. It looks more like a typo than a place to fill with other values.

<img width="237" height="44" alt="image" src="https://github.com/user-attachments/assets/1f328b98-8c20-4db3-877e-f782174cdec3" />

Reply:
The page below adds context to what this is though, in that use-case at least 🤔
In the current use case we might want to add `like useBarSeries for example`